### PR TITLE
Readme file

### LINKED ISSUE #23: Acceptance test fails due to lack of privilege for API key
```
=== RUN   TestAccDataSourceMongoDBAtlasProject_basic
--- FAIL: TestAccDataSourceMongoDBAtlasProject_basic (0.18s)
    testing.go:568: Step 0 error: errors during apply:

        Error: error creating project: POST https://cloud.mongodb.com/api/atlas/v1.0/groups: 401 (request "Unauthorized") The currently logged in user does not have the group creator role in organization 5b71ff2f96e82120d0aaec14.

          on C:\Users\SPENCE~1\AppData\Local\Temp\tf-test212583801\main.tf line 2:
          (source code not available)


=== RUN   TestAccDataSourceMongoDBAtlasProjects_basic
--- FAIL: TestAccDataSourceMongoDBAtlasProjects_basic (0.42s)
    testing.go:568: Step 0 error: errors during apply:

        Error: error creating project: POST https://cloud.mongodb.com/api/atlas/v1.0/groups: 401 (request "Unauthorized") The currently logged in user does not have the group creator role in organization 5b71ff2f96e82120d0aaec14.

          on C:\Users\SPENCE~1\AppData\Local\Temp\tf-test898250771\main.tf line 2:
          (source code not available)
```

Please document in the README the permissions required for the programmatic API key used in the acceptance tests.

### PR DESCRIPTION
**Pull request by PacoDw**
closes #23 #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,19 +47,19 @@ For either installation method, documentation about the provider specific config
 
 # Testing the Provider
 
-In order to test the provider, you can run `make test`. You need to [generic `provider` arguments](https://www.terraform.io/docs/configuration/providers.html) (e.g.
-`alias` and `version`), the following arguments are supported in the MongoDB
+In order to test the provider, you can run `make test`. Along with [generic provider arguments](https://www.terraform.io/docs/configuration/providers.html) (e.g.
+`project_id` and `org_id`), the following arguments are supported in the MongoDB
```
Comment (SpencerBrown) [inline README.md]: I think "(e.g. `alias` and `version`)" was correct, those are generic provider arguments, right?

Comment (PacoDw) [reply README.md]: yes, I mean, in the terraform documentation I didn’t find anything about the term 'generic', but I suppose that all generic arguments are from the provider itself, but if you want change them to "(e.g. alias and version)" I change. 👍

Comment (SpencerBrown) [reply README.md]: ok

Comment (PacoDw) [reply README.md]: ok, Mayble we could change to:
```
In order to test the provider, you can run `make test`. You need to meta-arguments such as 
`alias` and `version`, the following arguments are supported in the MongoDB.
```

what do you think?

**Comment by SpencerBrown** [comment]
Please be specific about what privileges are required for the programmatic API key used to run the acceptance tests.

**Comment by SpencerBrown** [comment]
The latest change says that Project Owner is required for the programmatic API key for the acceptance tests.

This cannot be right as my key did have Project Owner.

As the failure I documented was attempting to create a project, I suspect the key must have Project Creator permission at the Organization level. I will add this permission and retry the acceptance test.

**Pull request merged**